linkis-computation-governance/linkis-manager/linkis-application-manager/src/main/scala/org/apache/linkis/manager/am/service/engine/DefaultEngineCreateService.scala

```diff
@@ -293,6 +293,13 @@ class DefaultEngineCreateService
       })
     }
 
+    val queueRuleSuffix = props.get("queueRuleSuffix")
+    if (StringUtils.isNotBlank(queueRuleSuffix)) {
+      logger.info("Switch queues according to queueRule")
+      val queueName = props.getOrDefault("wds.linkis.rm.yarnqueue", "default")
+      props.put("wds.linkis.rm.yarnqueue", queueName + "_" + queueRuleSuffix)
```

Comment:
add log info 
logger.info(s"Switch queues according to queueRule with queue name :{queueName }  to ${newqueueName}")